articles/azure-app-configuration/concept-snapshots.md

```diff
@@ -12,6 +12,8 @@ ms.date: 05/16/2023
 
 A snapshot is a named, immutable subset of an App Configuration store's key-values. The key-values that make up a snapshot are chosen during creation time through the usage of key and label filters. Once a snapshot is created, the key-values within are guaranteed to remain unchanged.
 
+A brief overview is available in this [video](https://aka.ms/appconfig/snapshotVideo) which highlights three reasons that snapshots are an important tool in App Configuration.
```

Comment:
"important tool" made me feel the snapshot is only helpful for configuration management. It's useful in runtime, too. I am thinking of something like below (feel free to reword).

This video highlights three reasons why configuration snapshots can be helpful to you.

Reply:
Then how about

---

A brief overview is available in this [video](https://aka.ms/appconfig/snapshotVideo) which highlights three reasons that snapshots can be helpful to you.

Reply:
How do you like this (avoid nested attributive clauses)?

A brief overview is available in this [video](https://aka.ms/appconfig/snapshotVideo), highlighting three reasons that snapshots can be helpful to you.

Reply:
Sounds great !

Reply:
Updated.